Automattic/wp-calypso
# remove editable coming soon page for now

After discussion in pbxlJb-q6-p2 we've decided to deactivate the editable coming soon page for now

This PR takes it out of the editing toolkit.

### Testing instructions
sync this PR to your sandbox `yarn dev --sync`
Enable `define('WPCOM_PUBLIC_COMING_SOON', true)` in `0-sandbox.php`
Create a new site (wordpress.com/new?flags=coming-soon-v2)
It should not have a "coming soon" page in the pages list
Enter the new page workflow
Scroll to the bottom of the available page templates
There should no longer be a "coming soon" page template available
